onnxruntime/test/providers/qnn/qnn_basic_test.cc

```diff
@@ -1313,6 +1313,44 @@ TEST_F(QnnHTPBackendTests, DumpJsonQNNGraph) {
   std::filesystem::remove_all(dump_dir);
 }
 
+// Test exended UDMA mode on supported hardware (should run successfully)
```

Comment:
```suggestion
// Test extended UDMA mode on supported hardware (should run successfully)
```

how do we know if we are on supported hardware?